docs/VALIDATING_PERF.md

```diff
@@ -0,0 +1,45 @@
+# Validating Performance against Baseline
```

Comment:
Do you ever use this? I consider this feature heretic myself :-) Hunter should be used to find change points. If you fix all the regressions, then you don't  have regressions. If you don't, then you have regressions.

What this feature supports  is the old way of doing things: release candidate is slower than previous release but we don't know why. We then improve performance in an arbitrary component to compensate. When we get back to the original level, we can make a release.

Oh god now that I wrote it it sounds even worse! I suggest to deprecate this feature. It's existence misunderstands the whole benefit of hunter!

Reply:
We are not using this feature. I agree with you that it is not really helpful if one uses Hunter correctly. If you want, I can omit this feature in the docs to avoid potential increase of its user base :)

Reply:
Maybe remove it for now and I can start a email thread whether we want to deprecate it.

Reply:
38f531f